src/import/codegen.ts

```diff
@@ -90,7 +96,10 @@ export function generateConstruct(typegen: TypeGenerator, def: ApiObjectDefiniti
       delete copy['x-kubernetes-group-version-kind'];
 
       copy.required = copy.required || [];
-      copy.required = copy.required.filter(x => x !== 'apiVersion' && x !== 'kind' && x !== 'status');
+
+      if (Array.isArray(copy.required)) {
```

Comment:
This is just because `@types/json-schema` got bumped (breaking patch version...go figure)